predelete/predelete.go

```diff
@@ -62,6 +67,33 @@ func PreDeleteOperatorSecret(clientset *kubernetes.Clientset) error {
 	return nil
 }
 
+func PreDeleteEnterprisePortalSecret(clientset *kubernetes.Clientset) error {
+	fmt.Println("Running pre delete hook")
+	secrets, err := clientset.CoreV1().Secrets(os.Getenv("TYK_POD_NAMESPACE")).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	found := false
+	for _, value := range secrets.Items {
+		if value.Name == os.Getenv("ENTERPRISE_PORTAL_SECRET_NAME") {
```

Comment:
Should we replace hardcoded name for enterprise secret with constant value?
```suggestion
		if value.Name == os.Getenv(constants.EnterprisePortalSecretNameEnvVar) {
```

Or, if you do not need to re-read environment variable from the OS, we can use
```go
if value.Name == data.AppConfig.EnterprisePortalSecretName
```

Reply:
:upside_down_face: